CMakeLists.txt

```diff
@@ -20,30 +20,35 @@
 
 cmake_minimum_required(VERSION 3.5)
 
+include(GNUInstallDirs)
+
 add_subdirectory(media_driver)
 
 if (DEFINED BUILD_ALONG_WITH_CMRTLIB)
     add_subdirectory(cmrtlib)
 endif()
 
 if (TARGET iHD_drv_video AND TARGET igfxcmrt)
-    install (FILES ${CMAKE_CURRENT_BINARY_DIR}/media_driver/iHD_drv_video.so DESTINATION /usr/lib/x86_64-linux-gnu/dri/)
-    install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh  DESTINATION /etc/profile.d/)
-    install (FILES ${CMAKE_CURRENT_BINARY_DIR}/cmrtlib/linux/igfxcmrt64.so DESTINATION /usr/lib/x86_64-linux-gnu/)
+
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh.in ${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh)
+
+    install (FILES ${CMAKE_CURRENT_BINARY_DIR}/media_driver/iHD_drv_video.so DESTINATION ${CMAKE_INSTALL_LIBDIR}/dri/)
```

Comment:
on Ubuntu 64-bit, this CMAKE_INSTALL_LIBDIR will be 'lib', and the final install path will be '/usr/lib/dri/', which is not we wanted. how to remain original path (/usr/lib/x86_64-linux-gnu/dri/)?

Reply:
If you define `-DCMAKE_INSTALL_PREFIX=/usr` during `cmake`, then CMAKE_INSTALL_LIBDIR will be set to `/usr/lib/x86_64-linux-gnu` on Ubuntu.

Reply:
Also, user can override with `-DCMAKE_INSTALL_LIBDIR=<somedir>`